examples/main.rs

```diff
@@ -26,7 +26,7 @@ extern crate webkit2gtk;
 
 #[cfg(feature = "v2_4")]
 use glib::ToVariant;
-use gtk::{Inhibit, Window, WindowType, prelude::{ContainerExt, WidgetExt}};
+use gtk::{Inhibit, Window, WindowType, prelude::*};
 use webkit2gtk::{traits::{SettingsExt, WebContextExt, WebViewExt}, WebContext, WebView, WebViewExtManual};
```

Comment:
same for here, just use prelude instead of using traits directly

Reply:
as far as I can see `webkit2gtk-rs` doesn't have a prelude mod that re-exports everything.